src/App.vue

```diff
@@ -12,6 +11,12 @@
 
   <ButtonExample />
 
+
+  <DataTable :headers="tabelHeaders" :items="tabelContent" :paginationConfig ="paginationConfig"
+    @unsort="unsort" 
+    @sort-asc="sortAsc"
+    @sort-desc="sortDesc"
```

Comment:
another POV: what do you think about adding sorting functions on tableheaders configuration ? let's understanding that , firstly we change the name of tableHeaders simply to 'columns' for indicating the set of columns which be used, than we have already the cell formatter, i think we can also add the sorting events for each columns and to be more significative we must add a hint to indicating which column we had the sort ! (let's talk about it)